space-cli/modules/ingress/actions.go

```diff
@@ -7,6 +7,27 @@ import (
 	"github.com/urfave/cli"
 )
 
+// ActionGetIngressRoutes gets routes
+func ActionGetIngressRoutes(c *cli.Context) error {
+	// Get the project and url parameters
+	project := c.GlobalString("project")
+	commandName := c.Command.Name
+
+	params := map[string]string{}
+	if len(c.Args()) != 0 {
+		params["routesID"] = c.Args()[0]
```

Comment:
```suggestion
		params["id"] = c.Args()[0]
```

Reply:
This isn't my code. Wont be making changes. You make it in your PR